be/src/exec/hdfs_scanner.cpp

```diff
@@ -627,4 +628,12 @@ void HdfsScanner::move_split_tasks(std::vector<pipeline::ScanSplitContextPtr>* s
     }
 }
 
+CompressionTypePB HdfsScanner::get_compression_type_from_path(const std::string& filename) {
```

Comment:
Duplicate with the function return_compression_type_from_filename of hdf_scanner_text.cpp ?

Make it to one common function?